src/core_plugins/tests_bundle/tests_entry_template.js

```diff
@@ -26,7 +26,7 @@ window.__KBN__ = {
   vars: {
     kbnIndex: '.kibana',
     esShardTimeout: 1500,
-    esApiVersion: '2.0',
+    esApiVersion: 'master',
```

Comment:
@jbudz Just ignore this comment chain for the sake of this PR.

@spalger Why do we need to hardcode the version in package.json?  The version requirement is there for `npm`, which we don't publish Kibana to.  We should be able to mark the package.json as `private` and remove the version entirely.